apps/ideaspace/src/components/common/IdeaCard/IdeaCard.js

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react';
+import Link from 'next/link';
+import { atoms } from '@devlaunchers/components/src/components';
+import {
+  ImgButton,
+  StatuBox,
+} from './StyledIdeaCard';
+import commentSvg from '../../../images/comment.svg';
+import timeSvg from '../../../images/time.svg';
+
+function IdeaCard({ cards, cardType }) {
+  const [tag, setTag] = useState('');
+  const [tagColor, setTagColor] = useState('');
+  const [updated, setUpdated] = useState('');
+  React.useEffect(() => {
+    if (cards.statu == "submitted"){
+      setTag("submitted");
+      setTagColor("#A7D5E3");
+    } else if (cards.statu == "applying"){
+      setTag("applying");
+      setTagColor("#FFC44C");
+    } else if (cards.statu == "approved"){
+      setTag("approved");
+      setTagColor("#8CC882");
+    } else if (cards.statu == "archived"){
+      setTag("archived");
+      setTagColor("#75A3C0");
+    } else {
+      setTag("workshopping");
+      setTagColor("#FFA556");
+    }
+
+    const difftime = (new Date() - new Date(cards.updated_at)) / 1000;
+
+    if (difftime > (86400 * 30 * 12)) {
```

Comment:
Instead of using a "magic number" here, could we extract these numbers to variables? E.g.

`const secondsInDay = 86400;
const secondsInMonth = secondsInDay * 30`

https://en.wikipedia.org/wiki/Magic_number_(programming)
https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

Reply:
sure